index.ts

```diff
@@ -208,6 +208,8 @@ namespace logger {
       req.transaction.end();
     });
 
+    res.setHeader('X-FC-Transaction', req.transaction.id);
```

Comment:
I believe not. It's just sending headers. An issue would happen if you start sending the response body and then try to send a header.